imageprocessor/templates/input.html

```diff
@@ -1,3 +1,6 @@
+{% if errors %}
+   <p>We're sorry, but we couldn't process your image, please try a different file</p>
```

Comment:
may want to look into using django messages instead of hardcoding the error message in the template.
https://docs.djangoproject.com/en/2.1/ref/contrib/messages/

Reply:
Nice, I updated my PR to use the messaging middleware. It's still not great, but it's better.